client/my-sites/checkout/checkout-thank-you/business-plan-details.jsx

```diff
@@ -46,6 +47,17 @@ const BusinessPlanDetails = ( { selectedSite, sitePlans, selectedFeature } ) =>
 					href={ '/themes/' + selectedSite.slug } />
 			}
 
+			{ ! selectedFeature && isEnabled( 'manage/plugins/upload' ) &&
+				<PurchaseDetail
+					icon="plugins"
+					title={ i18n.translate( 'Upload a Plugin' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **68** times:
`translate( 'Upload Plugin' )` *ES Score: 11*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).